sc.py

```diff
@@ -0,0 +1,42 @@
+from microcosm.opaque import Opaque
```

Comment:
What is `sc.py` supposed to mean?

Don't these `print` statements blow up `flake8`?